ruby/README.md

```diff
@@ -0,0 +1,15 @@
+## Installation
+
+```bash
+git clone https://github.com/algolia/api-clients-playground
+```
+
+```bash
+cd api-clients-playground/ruby && bundle install
```

Comment:
What is bundle? If is a requirement you need to specify it. Concerning the template of the readme, can you update it to look exacly like this: https://github.com/algolia/api-clients-playground/pull/12/files#diff-41be41fc0c21938ff1ba8b2fdfa6291c.